packages/relay/src/lib/eth.ts

```diff
@@ -332,9 +332,21 @@ export class EthImpl implements Eth {
     this.logger.trace(`${requestIdPrefix} estimateGas(transaction=${JSON.stringify(transaction)}, _blockParam=${_blockParam})`);
     //this checks whether this is a transfer transaction and not a contract function execution
     if (!transaction || !transaction.data || transaction.data === '0x') {
-      const toAccount = await this.mirrorNodeClient.getAccount(transaction.to);
+      const accountCacheKey = `account_${transaction.to}`;
+      let toAccount: object | null = this.cache.get(accountCacheKey);
```

Comment:
This is a cache addition not an improvement.
Please update the PR description with the goal and changes here